### DISCUSSION TITLE
enhmstratigraphy - Stratigraphy module

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#956] [Structural Changes]
This is a discussion to collate and discuss changes that we would like to see in the stratigraphy module in EMu. These may be structural or data.

[OPEN STRUCTURAL CHANGE ISSUES](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22structural+changes%22+label%3Aenhmstratigraphy)

**Comment by cmaves3** [comment]
A - In Biostrat. tab:
1) Add field named "Land Mammal Age" to Details section

B - In Chronostrat tab (pics):
1) For consistency in terms, the field "Stage" should be changed to "Age" 
2) The term Chronostrat would, in this use, be more accurate as Geochronologic

![Stratigraphy_Chronostrat](https://github.com/fieldmuseum/EMu-Documentation/assets/128521785/6fc15233-fab4-4a13-8305-92df59b35c2e)
![Chronostrat vs Geochronologic](https://github.com/fieldmuseum/EMu-Documentation/assets/128521785/875dcc95-f09b-4deb-8ffb-ab5c0bd00522)

**Comment by rondlg** [comment]
Updates to the way we handle name strings: 

[Discussion Topic](https://github.com/fieldmuseum/EMu-Documentation/discussions/1125)
